pkgs/top-level/aliases.nix

```diff
@@ -332,33 +332,34 @@ mapAliases ({
   pg_tmp = ephemeralpg; # added 2018-01-16
 
   php-embed = throw ''
-    php*-embed has been dropped, you can build the same package by using
-     something similar with this following snippet:
-    (php74.override { config.php.embed = true; config.php.apxs2 = false; })
+    php*-embed has been dropped, you can build the something similar
```

Comment:
this sentence seems wrong to me. I assume the `the` is superfluous?

Reply:
Yep, just forgot to remove the `the`. Thanks!